main.ps1

```diff
@@ -12,7 +12,7 @@ switch ($Type) {
    }
    'ModulePath' {
       if ($env:RUNNER_OS -eq "Windows") {
-         $modpath = ($env:PSModulePath.Split(";") | Select-Object -First 1)
+         $modpath = "$env:ProgramFiles\PowerShell\Modules"
          if ($Shell -eq "powershell") {
             return $modpath.Replace("PowerShell","WindowsPowerShell")
```

Comment:
ayyy, i was just about to ask about pwsh vs powershell compat but it looks like i took care of this earlier 😅